airflow/models/dagrun.py

```diff
@@ -141,7 +142,7 @@ class DagRun(Base, LoggingMixin):
     triggered_by = Column(
         Enum(DagRunTriggeredByType, native_enum=False, length=50)
     )  # Airflow component that triggered the run.
-    conf = Column(PickleType)
+    conf = Column(JSON().with_variant(postgresql.JSONB, "postgresql"))
```

Comment:
will this work for non-postgresql db?

Reply:
Yeah. I checked how this works. Looks like, only for postgres, this setting will be applied. Not for other databases